reference/forms/types/choice.rst

```diff
@@ -119,9 +119,19 @@ With this option you can also allow float values to be selected as data.
 
     // ...
     $builder->add('status', 'choice', array(
-      'choice_list' => new ChoiceList(array(1, 0.5), array('Full', 'Half'))
+      'choice_list' => new ChoiceList(array(1, 0.5, 0.1), array('Full', 'Half', 'Almost empty'))
     ));
 
+The ``status`` field created by the code above will be rendered as:
+
+.. code-block:: html
+
+    <select name="status">
+        <option value="0">Full</option>
+        <option value="1">Half</option>
+        <option value="2">Almost empty</option>
+    </select>
+
```

Comment:
This is a nice addition, because as I was reading this, I was in fact confused as you described in the PR description :). Can we add one more line after this? I'd like something that basically says: "But don't be confused! If ``Full`` is selected (value 0 in HTML), 1 will be returned in your form. If ``Almost empty`` is selected (value 2 in HTML), 2 will be returned".